types/sync_aggregate.yaml

```diff
@@ -0,0 +1,11 @@
+SyncAggregate:
+  type: object
+  description: "The [`SyncAggregate`](https://github.com/ethereum/eth2.0-specs/blob/dev/specs/altair/beacon-chain.md#syncaggregate) object from the Eth2.0 Altair spec."
```

Comment:
It should be linked with spec tag or commit to prevent repo reorg break our links